compiler/src/dotty/tools/dotc/core/tasty/TastyFormat.scala

```diff
@@ -416,6 +416,7 @@ object TastyFormat {
     METHODtype + implicitOffset + erasedOffset
   }
 
+  final val APPLIEDTERMREF = 180
```

Comment:
Need to add this to the comment listing all tags at the top of this file and to bump the MinorVersion of TASTY. Also may need to add a case to  TastyPrinter (To check if TastyPrinter is working correctly, enable "pickling" in Printers.scala)